Makefile

```diff
@@ -28,10 +28,10 @@ ifeq (${CI}, true)
 endif
 
 # union for 'make test'
-UNION := functional debug-console $(DOCKER_DEPENDENCY) openshift crio docker-compose network \
-	docker-stability oci netmon kubernetes swarm vm-factory \
-	entropy ramdisk shimv2 tracing time-drift compatibility vcpus \
-	$(PODMAN_DEPENDENCY) pmem
+UNION := crio compatibility debug-console $(DOCKER_DEPENDENCY) docker-compose \
```

Comment:
lgtm
if we make this a 'one item per line' list then I think the ordering would be even more obvious. wdyt @jodh-intel